.circleci/config.yml

```diff
@@ -28,8 +30,9 @@ jobs:
       # Configure the JVM and Gradle to avoid OOM errors
       _JAVA_OPTIONS: "-Xmx1g"
       GRADLE_OPTS: "-Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2"
+      ORG_GRADLE_PROJECT_testAdditionalJavaVersions: true
     docker:
-      - image: circleci/openjdk:8-jdk
+      - image: circleci/openjdk:11.0.6-jdk-buster
```

Comment:
Are we sure that this image is being used? The line 25 defines executor type as machine which is VM and I think as it is applied at root all jobs will use it.

Reply:
Right I found out it's not being used. I'm going to be deleting CircleCI build anyways :)